src/test/java/uk/gov/pay/adminusers/resources/ServiceResourceCreateIT.java

```diff
@@ -1,37 +1,117 @@
 package uk.gov.pay.adminusers.resources;
 
-import com.fasterxml.jackson.databind.JsonNode;
+import io.restassured.response.ValidatableResponse;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
 import org.junit.jupiter.api.Test;
-import uk.gov.service.payments.commons.model.SupportedLanguage;
 
 import java.util.List;
 import java.util.Map;
 
 import static io.restassured.http.ContentType.JSON;
+import static org.hamcrest.Matchers.emptyIterable;
 import static org.hamcrest.Matchers.hasEntry;
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.not;
 import static org.hamcrest.Matchers.nullValue;
+import static uk.gov.service.payments.commons.model.SupportedLanguage.ENGLISH;
+import static uk.gov.service.payments.commons.model.SupportedLanguage.WELSH;
 
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
 class ServiceResourceCreateIT extends IntegrationTest {
 
     @Test
-    void shouldCreateService() {
-        JsonNode payload = mapper
-                .valueToTree(Map.of(
-                        "service_name", Map.of(SupportedLanguage.ENGLISH.toString(), "Service name"),
-                        "gateway_account_ids", List.of("1")
-                ));
-        
-        givenSetup()
+    void create_service_with_all_parameters_successfully() {
+        var validatableResponse = givenSetup()
                 .when()
                 .contentType(JSON)
-                .body(payload)
+                .body(Map.of("service_name", Map.of(ENGLISH.toString(), "Service name"),
+                        "gateway_account_ids", List.of("1")))
                 .post("v1/api/services")
                 .then()
                 .statusCode(201)
                 .body("created_date", is(not(nullValue())))
                 .body("gateway_account_ids", is(List.of("1")))
                 .body("service_name", hasEntry("en", "Service name"));
+        
+        assertStandardFields(validatableResponse);
+    }
+
+    private void assertStandardFields(ValidatableResponse validatableResponse) {
+        validatableResponse
+                .body("current_psp_test_account_stage", is("NOT_STARTED"))
+                .body("current_go_live_stage", is("NOT_STARTED"))
+                .body("default_billing_address_country", is("GB"))
+                .body("agent_initiated_moto_enabled", is(false))
+                .body("takes_payments_over_phone", is(false))
+                .body("experimental_features_enabled", is(false))
+                .body("internal", is(false))
+                .body("archived", is(false))
+                .body("redirect_to_service_immediately_on_terminal_state", is(false))
+                .body("collect_billing_address", is(true));
+    }
```

Comment:
Maybe this method should go after all the tests that rely on it.

Reply:
How do you mean? Like for other integration tests?

Reply:
Or do you mean moving this method to the end of the class...

Reply:
Yeah, at the end of the class or at least after all the tests that rely on it.